changelog/v0.3.1/fix-resolve-deps.yaml

```diff
@@ -0,0 +1,7 @@
+changelog:
+  - type: FIX
```

Comment:
I think this should be a breaking change, as we are removing exported functions/types from `pkg`. Someone might depend on them. Wdyt?